### ISSUE TITLE
usefulness of `nbhs_pinfty_gt_pos`

### ISSUE THREAD

**Issue by affeldt-aist** [math-comp/analysis#486]
https://github.com/math-comp/analysis/blob/87030d669403157aff8dbb676fa901521a03a131/theories/normedtype.v#L518

Is `nbhs_pinfty_gt_pos` really necessary? Isn't it subsumed by `nbhs_pinfty_gt`:

https://github.com/math-comp/analysis/blob/87030d669403157aff8dbb676fa901521a03a131/theories/normedtype.v#L512

Note that both are actually used as of today.

**Comment by proux01** [comment]
We should probably try to replace uses of the first by the second and check that this is not too costly.

**Comment by affeldt-aist** [comment]
I think that I tried but was not convinced. Let me try again.

**Comment by affeldt-aist** [comment]
See PR #559
